### PR TITLE
Invite dialog with FB share link

### PR DESCRIPTION
**Pull request by imostovenko**

### CODE REVIEW

File: components/layout/side-drawer/side-drawer.layout.html [hunk 1]
```diff
@@ -4,19 +4,18 @@
...
-  style="top: 64px !important;"
+  style="top: 56px !important;"
 >
   <v-tabs v-model="selectedTab" grow class="pa-0">
     <v-tab key="service" href="#tab-mine">my plans</v-tab>
     <v-tab key="privacy" href="#tab-friends">friends plans</v-tab>
   </v-tabs>
 
-  <v-tabs-items v-model="selectedTab" style="height: calc(100% - 48px)">
+  <v-tabs-items v-model="selectedTab" style="height: calc(100% - 48px); background-color: #f5f5f6;">
```
Comment (acharlop) [inline components/layout/side-drawer/side-drawer.layout.html]: please move this to a class

File: components/dialogs/plan-form/plan-form.dialog.js [hunk 2]
```diff
@@ -20,7 +20,7 @@ export default Vue.component('PlanFormDialog', {
       description: '',
       startAtMenu: false,
       endAtMenu: false,
-      placeRules: [(v) => !!v || 'Name is required'],
+      placeRules: [(v) => !!v || 'Place is required'],
```
Comment (acharlop) [inline components/dialogs/plan-form/plan-form.dialog.js]: this was already fixed, it's going to cause conflicts. Please remove

File: components/dialogs/plan-form/plan-form.dialog.html [hunk 3]
```diff
@@ -88,7 +88,7 @@
                 :max="1"
                 ticks="always"
                 tick-size="4"
-                class="mb-5"
+                class="mb-5 pl-5 pr-5"
```
Comment (acharlop) [inline components/dialogs/plan-form/plan-form.dialog.html]: simpler `class="mb-5 px-5"`

File: components/dialogs/invites/invites.dialog.html [hunk 4]
```diff
@@ -15,25 +15,27 @@
...
-        </br>
-        Or simply share us on your Facebook timeline 
-      <!-- Share FB button-->
-        <button class="auth-button" data-href="https://nomad-plans.herokuapp.com/" >
-          <fa class="icon" :icon="icon" /> 
-          <div class="text"><a target="_blank" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fnomad-plans.herokuapp.com%2F&amp;src=sdkpreparse" >Share</a></div>
-        </button>
+        <!-- Share FB button-->
+        <p class="my-5">Share us on your Facebook timeline </p>
+        <p >
```
Comment (acharlop) [inline components/dialogs/invites/invites.dialog.html]: if it's buttons and not text it probably should be a `div`

File: components/layout/side-drawer/side-drawer.layout.html (hunk 1 above)
Comment (imostovenko) [reply components/layout/side-drawer/side-drawer.layout.html]: added to .side-drawer class in css

**Pull request merged**